python/tvm/autotvm/graph_tuner/utils/utils.py

```diff
@@ -73,7 +73,7 @@ def is_boundary_node(node_entry, input_names):
     # Operators dependent on original layouts.
     _LAYOUT_FIXED_OP = ["batch_flatten", "transpose", "reshape",
                         "multibox_prior", "multibox_transform_loc", "where",
-                        "non_max_suppression", "strided_slice"]
```

Comment:
Why do we want to change this?

Reply:
This was causing problem in quantized mobilenet conv2d. QNN inserts strided slice for quantized conv2d. This caused many strided slice, and somehow graph tuning dependency map generation was incorrect. We already have a stridedSliceInferLayout attritbute, so I don't think this is required. But, let me know, I dont fully understand the graph tuner dependency collection code yet. If you know, why this is here, I can also test the changes.

Reply:
After talking to Yao, we decided to solve it in a different manner. It might work for the case that I experienced with QNN, but might fail in other cases. We will revisit this.